vms/platformvm/txs/executor/camino_tx_executor_test.go

```diff
@@ -547,3 +547,266 @@ func TestCaminoAddValidatorTxNodeSig(t *testing.T) {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[2].PublicKey().Address(),
+			flag:          txs.AddressStateKycBits,
+			state:         txs.AddressStateRoleKyc,
+			expectedErr:   errInvalidRoles,
+			remove:        false,
+		},
+		"Flag: AddressStateMax Add": {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[0].PublicKey().Address(),
+			flag:          txs.AddressStateMax,
+			state:         txs.AddressStateRoleAdmin,
+			expectedErr:   nil,
+			remove:        false,
+		},
+		"Flag: AddressStateMax Remove": {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[0].PublicKey().Address(),
+			flag:          txs.AddressStateMax,
+			state:         txs.AddressStateRoleAdmin,
+			expectedErr:   nil,
+			remove:        true,
+		},
+		"Flag: AddressStateValidBits Add": {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[0].PublicKey().Address(),
+			flag:          txs.AddressStateValidBits,
+			state:         txs.AddressStateRoleValidator,
+			expectedErr:   nil,
+			remove:        false,
+		},
+		"Flag: AddressStateValidBits Remove": {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[0].PublicKey().Address(),
+			flag:          txs.AddressStateValidBits,
+			state:         txs.AddressStateRoleValidator,
+			expectedErr:   nil,
+			remove:        true,
+		},
+		"Flag: AddressStateRoleAdmin Add, State: AddressStateKycExpired": {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[0].PublicKey().Address(),
+			flag:          uint64(txs.AddressStateRoleAdmin),
+			state:         txs.AddressStateKycExpired,
+			expectedErr:   errInvalidRoles,
+			remove:        false,
+		},
+		"Flag: AddressStateRoleKyc Add, State: AddressStateRoleValidatorBit": {
+			stateAddress:  caminoPreFundedKeys[0].PublicKey().Address(),
+			targetAddress: caminoPreFundedKeys[0].PublicKey().Address(),
+			flag:          uint64(txs.AddressStateRoleKyc),
+			state:         uint8(txs.AddressStateRoleValidatorBit),
+			expectedErr:   txs.ErrInvalidState,
+			remove:        false,
+		},
+		"Wrong address": {
+			stateAddress:  ids.GenerateTestShortID(),
+			targetAddress: ids.GenerateTestShortID(),
+			flag:          uint64(txs.AddressStateRoleKyc),
+			state:         txs.AddressStateRoleKyc,
+			expectedErr:   errInvalidRoles,
+			remove:        false,
+		},
+		"Empty address": {
+			stateAddress:  ids.ShortEmpty,
+			targetAddress: ids.ShortEmpty,
+			flag:          uint64(txs.AddressStateRoleKyc),
+			state:         txs.AddressStateRoleKyc,
+			expectedErr:   txs.ErrEmptyAddress,
+			remove:        false,
+		},
+	}
+
+	generateExecutor := func(unsidngedTx txs.UnsignedTx, env *environment) CaminoStandardTxExecutor {
```

Comment:
there is no point in making this func, imo
u'll still use it in only one place

Reply:
OK, removed